scripts/zkai_one_block_executable_package_accounting_gate.py

```
@@ -0,0 +1,660 @@
+def _int(value: Any, field: str) -> int:
+    if not isinstance(value, int) or isinstance(value, bool):
+        raise OneBlockPackageAccountingError(f"{field} must be integer")
+    return value
+
+
+def load_json(path: pathlib.Path) -> tuple[dict[str, Any], bytes]:
+    try:
+        raw = SURFACE.read_source_bytes(path, str(path.relative_to(ROOT)))
+        payload = json.loads(raw.decode("utf-8"), parse_constant=_reject_json_constant)
+    except Exception as err:
+        raise OneBlockPackageAccountingError(f"failed to load source evidence {path}: {err}") from err
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)